features/step_definitions/visual_editor_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^The visual editor private beta feature flag is (enabled|disabled)$/) do |enabled|
+  @test_strategy ||= Flipflop::FeatureSet.current.test!
+  @test_strategy.switch!(:govspeak_visual_editor, enabled == "enabled")
+end
+
+When(/^I start creating a new publication$/) do
+  visit new_admin_publication_path
+end
+
+Then(/^I should see the visual editor instead of the govspeak editor$/) do
+  expect(page).to have_selector(".app-c-visual-editor__container")
+end
+
+Then(/^I should see the govspeak editor instead of the visual editor$/) do
+  expect(page).to have_selector(".app-c-govspeak-editor")
+end
+
+When(/^I start creating a new HTML attachment for a publication$/) do
+  publication = create(:publication)
+  visit new_admin_edition_attachment_path(publication, type: "html")
+end
+
+Given(/^a draft publication with an HTML attachment "(.*?)" exists$/) do |title|
+  html_attachment = create(:html_attachment, title:)
+  @publication = create(:publication, attachments: [html_attachment])
+end
+
+When(/^I edit the HTML attachment "(.*?)"$/) do |title|
+  html_attachment = Attachment.find_by(title:)
```

Comment:
I know we can't set the ID ourselves, but perhaps we could use the base path or something that we can be confident in the format of? Matching on strings with no specified format is just one of those things I try to avoid wherever possible, even though it's probably perfectly safe in this case

Reply:
We can always use a `@html_attachment` and not include the title at all. It seemed a pattern in other tests, in the sense that it reads well.